pkg/rpc/response/result/unclaimed.go

```diff
@@ -0,0 +1,46 @@
+package result
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// GASID stores GAS asset ID
+const GASID = "602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7"
+
+// Unclaimed wrapper is used to represent getunclaimed return result.
+type Unclaimed struct {
+	Available   util.Fixed8 `json:"available"`
+	Unavailable util.Fixed8 `json:"unavailable"`
+	Unclaimed   util.Fixed8 `json:"unclaimed"`
+}
+
+// NewUnclaimed creates a new Unclaimed wrapper using given Blockchainer.
+func NewUnclaimed(a *state.Account, chain core.Blockchainer) (Unclaimed, error) {
+	var (
+		available   util.Fixed8
+		unavailable util.Fixed8
+	)
+
+	for _, ub := range a.Unclaimed {
+		gen, sys, err := chain.CalculateClaimable(ub.Value, ub.Start, ub.End)
+		if err != nil {
+			return Unclaimed{}, err
+		}
+		available += gen + sys
+	}
+
+	balanceValues := a.GetBalanceValues()
+	gasID, err := util.Uint256DecodeStringLE(GASID)
+	if err != nil {
+		return Unclaimed{}, err
+	}
+	unavailable = balanceValues[gasID]
```

Comment:
That's not correct, you need to get all NEO UTXO's and make `CalculateClaimable()` for them using current blockchain height as an end value.